src/bartholomew.rs

```diff
@@ -50,6 +50,16 @@ pub fn render(req: Request) -> Result<Response> {
         config.base_url = Some(url);
     }
     eprintln!("Base URL: {:?}", &config.base_url);
+    eprintln!("Prepend route info : {:?}", &config.prepend_route_info);
+
+    if let Some(true) = config.prepend_route_info {
+        let route_info = match req.headers().get("spin-component-route") {
+            Some(route) => route.to_str().unwrap_or_default(),
+            None => "",
+        };
+        path_info = format!("{route_info}{path_info}");
+        eprintln!("Updated request path: {:?}", path_info);
```

Comment:
This may be a lost cause but this is going to triple the amount of console spew.  Would it be less punishing to assemble the three logs into one e.g. "Resolved path: {:?} (from base {:?} and prepend={}"?  (I am not saying it would be... just prompting for your thoughts.)

Reply:
That sounds like a good idea. Also this does exactly triple the console spew as there are other places where we write to console when handling a request. I will update it in any case.